CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Changelog
 ## vNEXT
-
+### Highlights :tada:
++ Add support for bracket syntax to facades. The annotation @PyBracketAccess can be used on methods to mark them as representing bracket access on an object. The target method must have one (to read the value) or two parameters (to update the value).
```

Comment:
Need backticks around `@PyBracketAccess` to format correctly. Also, please refer to the PR at the end of the line like in earlier changelog items.